data-raw/canopy/canopy.R

```diff
@@ -4,6 +4,9 @@ nlcd_based = read.table('data-raw/canopy/hc_sheltering.tsv', as.is=TRUE, sep='\t
 names(nlcd_based) = c('site_id', 'canopy_m')
 nlcd_based$source = 'nlcd'
 
-canopy = rbind(nlcd_based)
+# get new data from expanded footprint
+new_canopy <- readRDS('data-raw/canopy/lakeattributes_canopyheight.rds')
+
+canopy = rbind(nlcd_based, new_canopy)
```

Comment:
do we need a `distinct()` here like in the other updates? or are there no duplicates for the expanded footprint?

Reply:
There *shouldn't* be...but isn't a bad idea.